l10n_it_central_journal/i18n/it.po

```diff
@@ -306,3 +306,10 @@ msgstr "report.l10n_it_central_journal.report_giornale"
 #: model:ir.model,name:l10n_it_central_journal.model_wizard_giornale
 msgid "wizard.giornale"
 msgstr "Wizard libro giornale"
+
```

Comment:
please do not edit `*.po*` files, they might conflict with Weblate (see https://odoo-community.org/groups/contributors-15/contributors-80677?mode=thread&date_begin=&date_end=)